components/CountryEditForm/index.js

```diff
@@ -0,0 +1,105 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function CountryEditForm({
+  country,
+  onSubmitEdit,
+  handleInputChangeName,
+  onEdit,
+  handleInputChangeStartDate,
+  handleInputChangeEndDate,
+}) {
+  if (country === undefined) {
+    return null;
+  }
+  return (
+    <>
+      <form onSubmit={onSubmitEdit}>
+        <StyledFieldset>
+          <StyledDiv>
+            <StyledLabel htmlFor="name">Land hinzufügen: </StyledLabel>
```

Comment:
I would change the label to "Land:" or something, because you don't really want to add something, you just want to edit.